opensearch-cluster-cdk/README.md

```diff
@@ -0,0 +1,66 @@
+# CDK for deploying single-node and multi-node OpenSearch cluster with dashboards
+
+This project enables uses to deploy either a single-node or a multi-node OpenSearch cluster.
+There are two stacks that get deployed:
+1. OpenSearch-Network-Stack: Use this stack to either use an existing Vpc or create a new Vpc. This stack also creates a new security group to manage access.
+2. OpenSearch-Infra-Stack: Sets up EC2 ASG (installs opensearch and opensearch-dashboards using userdata), cloudwatch logging, load balancer. Check your cluster log in the log group created from your stack in the cloudwatch.
```

Comment:
Can you also specify what kind of AMI (AL2/ubuntu/etc) we are using in this? Seeing a lot of `/home/ec2-user` commands so if the user uses something else, its gonna fail. Or maybe replace those commands with someone `$(whoami)`. Can come as an enhancement